epictrack-api/migrations/versions/735a693b4a59_fix_to_existing_event_configurations.py

```diff
@@ -0,0 +1,31 @@
+"""fix_to_existing_event_configurations
+
+Revision ID: 735a693b4a59
+Revises: 5b390f888c3f
+Create Date: 2024-02-09 00:34:48.510254
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '735a693b4a59'
+down_revision = '5b390f888c3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("update event_configurations ec set event_position = 'START' where ec.template_id in (select id from event_templates et where et.phase_id = 20 and name = 'Referral Package Received by Ministers' order by sort_order)")
+    op.execute("update event_configurations ec set event_position = 'END' where ec.template_id in (select id from event_templates et where et.phase_id = 20 and name = 'EAC Ministers Decision' order by sort_order)")
```

Comment:
Could Phase ID differ depending on the order we import event templates?

Reply:
That is a possibility. I will put another nested query for this.